src/palette/view/widgets/timedialog.h

```diff
@@ -70,4 +70,4 @@ private slots:
 };
 }
 
-#endif
+#endif
```

Comment:
Another unnecessary whitespace change: you removed the newline at the end of the file

Reply:
Extremely sorry as this was my first time working over here messed up a little bit. I will quickly sort it out and create a similar PR